tests/base/__init__.py

```diff
@@ -43,6 +44,19 @@ def override_config(self, integration, values):
         finally:
             options.update(original)
 
+    @contextlib.contextmanager
+    def override_global_tracer(self):
+        """
+        Helper functions that overrides the global tracer available in the
+        `ddtrace` package. This is required because in some `httplib` tests we
+        can't get easily the PIN object attached to the `HTTPConnection` to
+        replace the used tracer with a dummy tracer.
+        """
+        original_tracer = ddtrace.tracer
+        ddtrace.tracer = self.tracer
```

Comment:
I just ran into this in another test. This only works if the test does `import ddtrace` and uses `ddtrace.tracer`

If however the test (or module we are testing) does `from ddtrace import tracer` then this helper won't work.